pasos-para-investigar.md

```diff
@@ -0,0 +1,51 @@
+## Pasos para Investigar
+
+1. **Recopilar información (datos científicos, libros u otras fuentes confiables)**
+2. **Entender la taxonomía de la planta**
+	- Dominio
+	- Reino
+	- Filo o división
+	- Clase
+	- Orden
+	- Familia
+	- Género
+	- Especie
+
+Es importante entender la taxonomía para realizar la investigación con un enfoque claro de lo que estamos buscando.
+
+3. **Organizar la información en base a los siguientes puntos:**
+Brief intro. Abstract or summary.
+
+- **Botanical characteristics** 
+	- Family or families
+	- Categories known
+	- Origin - climatic and soil characteristics of the region of origin
+	- Adaptation / Mutation / Engineering - adaption to different climatic and soil characteristics of the regions where currently cultivated; natural mutation known due to adaptation (if any); changes due to engineering of the plant.
+	- Physical characteristics – general description supported by a picture gallery, displaying such characteristics
+- **Properties**
+	- Scientifically researched or proven properties
+	- Folklore or traditions – properties communities have attributed to the plant or plant family over generations. These are facts or beliefs known to the communities, which may or may not have been scientifically proven.
+	- Local traditions, uses, beliefs – properties and or uses the local people of the region (could be the country) have attributed to the plan
+- **Current known uses:**
+	- Health / pharmacological
+	- Food chain (health)
+	- Cosmetic / chemical
+	- Ornamental
+- **Additional observations**
+ 	- Here we point out to non-verified claims, myths, or confusions that may exists.  For example, refereeing to the plant with a wrong name (categorizing it improperly) because of how it looks.
+	- History of the plant in the Dominican Republic
+	- What is known or believed?
+	- How did it arrive to the island and when?
+	- Where does it growth and/or how is it cultivated (indicating the various uses).
+
+- **Resumir. Evitar repeticiones y resaltar puntos clave.**
+- **Releer y corregir información.**
+- **Localizar conceptos clave e investigar su significado para incluirlos en el glosario.**
+
+##List of trusted sources##
+- [Everipedia](https://everipedia.org/)
+- [Kew](https://www.kew.org/)
+- [Scielo](https://scielo.org/en/)
+- [Encyclopedia Britannica](https://www.britannica.com/)
+- [Science Direct](https://www.sciencedirect.com/)
+
```

Comment:
If we're going to use bold for the titles, it would aid readability to use a header element. Example
```
# Pasos para Investigar
_Aquí una breve descripción del documento y su uso_

## 1. Recopilar información (datos científicos, libros u otras fuentes confiables)

## 2. Entender la taxonomía de la planta
```

# Pasos para Investigar
_Aquí una breve descripción del documento y su uso_

## 1. Recopilar información (datos científicos, libros u otras fuentes confiables)

## 2. Entender la taxonomía de la planta